.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
You made some changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/javascript)
**In the next project, kindly use the linter config from Microverse 👆**  
![image](https://user-images.githubusercontent.com/31897434/73417568-ca44bd00-4318-11ea-8a5e-fedabd629b8c.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/